pom.xml

```diff
@@ -34,6 +35,15 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+            <scope>runtime</scope>
```

Comment:
incorrect scope - it should be `compile` and since `compile` is the default scope - you may omit specifying scope at all


I'm not even sure how it can work for you with `runtime` scope 0_o